src/client/utils/validations.js

```diff
@@ -333,19 +336,20 @@ function isValidChildrenIncome(children) {
 
 function isValidAnnualIncome(data) {
   let isValidSpouseIncomeFields = true;
-
-  if (data.spouseGrossIncome && data.spouseNetIncome && data.spouseOtherIncome) {
+  if (data.maritalStatus.value === 'Married' || data.maritalStatus.value === 'Separated') {
     isValidSpouseIncomeFields =
-      isValidField(isValidMonetaryValue, data.spouseGrossIncome) &&
-      isValidField(isValidMonetaryValue, data.spouseNetIncome) &&
-      isValidField(isValidMonetaryValue, data.spouseOtherIncome);
+      isValidRequiredField(isValidMonetaryValue, data.spouseGrossIncome) &&
+      isValidRequiredField(isValidMonetaryValue, data.spouseNetIncome) &&
+      isValidRequiredField(isValidMonetaryValue, data.spouseOtherIncome);
   }
 
-  return isValidField(isValidMonetaryValue, data.veteranGrossIncome) &&
-    isValidField(isValidMonetaryValue, data.veteranNetIncome) &&
-    isValidField(isValidMonetaryValue, data.veteranOtherIncome) &&
-    isValidSpouseIncomeFields &&
-    isValidChildrenIncome(data.children);
+  return data.understandsFinancialDisclosure || (
```

Comment:
I think this should be checking for `data.understandsFinancialDisclosure === N` (same thing elsewhere)